core/src/test/java/io/aiven/kafka/tieredstorage/chunkmanager/cache/ChunkCacheTest.java

```diff
@@ -213,7 +213,7 @@ void prefetchingNextChunk() throws Exception {
             chunkCache.configure(Map.of(
                 "retention.ms", "-1",
                 "size", "-1",
-                "prefetching.size", "1"
+                "prefetching.bytes", "10"
```

Comment:
It would be nice to bind this `10` to some named constant

Reply:
Fixed